uniflow/flow/config.py

```diff
@@ -40,6 +40,20 @@ class ExtractPDFConfig(ExtractConfig):
     model_config: ModelConfig = NougatModelConfig()
 
 
+@dataclass
+class ExtractMarkdownConfig(ExtractConfig):
+    """Extract markdown Config Class."""
+
+    flow_name: str = "ExtractMarkdownFlow"
```

Comment:
qq: do we need to include markdown related code in this PR? It looks like it is already included in https://github.com/CambioML/uniflow/pull/71